charts/k8ssandra/values.yaml

```diff
@@ -27,6 +27,19 @@ cassandra:
   #
   # image:
 
+  # -- The image registry to use for all images deployed as part of the
+  # CassandraDatacenter.
+  imageRegistry: docker.io
+
+  # -- The server-config-init init container image
+  configBuilderImage: datastax/cass-config-builder:1.0.4
+
+  # -- The server-system-logger container image
+  systemLoggerImage: k8ssandra/system-logger:9c4c3692
```

Comment:
Are we waiting for an official system-logger image to be published, or is this the hash tag here just temporary??

Reply:
This is currently the image used in cass-operator. I think cass-operator needs to be updated to build tagged versions of system-logger.

Reply:
What is this image? Is it just `busybox` and `tini`?

Reply:
yes